ui/lib/components/AutoRefreshButton/AutoRefreshButton.tsx

```diff
@@ -133,11 +135,23 @@ export function AutoRefreshButton({
 
   return (
     <Dropdown.Button
+      className={styles.auto_refresh_btn}
       disabled={disabled}
       onClick={handleRefresh}
       overlay={autoRefreshMenu}
       trigger={['click']}
-      icon={<DownOutlined />}
+      icon={
+        <>
+          {autoRefreshSeconds ? (
```

Comment:
Maybe you should move the circle ring to here.

Reply:
But I feel like it looks more balanced now and closer to grafana.

![image](https://user-images.githubusercontent.com/11549583/148316378-ba73e97e-599c-4569-9f39-4081215971f2.png)
![image](https://user-images.githubusercontent.com/11549583/148316491-07bbf7a9-3b69-4aa1-b692-7580d83e85f2.png)
![image](https://user-images.githubusercontent.com/11549583/148316541-135426ba-4717-4237-95eb-3c1bff1c1096.png)